wos/get_pubs.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+import requests
+import xml.etree.cElementTree as ET
+import yaml
+
+def run(wosnnection, author, categories):
```

Comment:
Take another look and see if there is something that is a little less string manipulation-y for this.